src/api/app/views/webui2/webui/comment/_delete_dialog.html.haml

```diff
@@ -0,0 +1,13 @@
+.modal.fade{ id: "delete-comment-modal-#{comment.id}", tabindex: -1, role: 'dialog',
+             aria: { labelledby: 'delete-comment-modal-label', hidden: true } }
+  .modal-dialog{ role: 'document' }
+    .modal-content
+      .modal-header
+        %h5.modal-title
+          Do you really want delete this comment?
+      .modal-body
+        %p Please confirm deletion of comment
```

Comment:
I wonder if we should include the comment here? It might be a bit long though... what do you think?

Reply:
Okay fine